tests/Integration/inc/Engine/Media/PreloadFonts/Frontend/Subscriber/maybeRemoveExistingPreloadedFonts.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+
+namespace WP_Rocket\Tests\Integration\inc\Media\PreloadFonts\Frontend\Subscriber;
+
+use WP_Rocket\Tests\Integration\TestCase;
+
+/**
+ * Test class covering \WP_Rocket\Engine\Media\PreloadFonts\Frontend\Subscriber::get_exclusions
+ *
+ * @group PerformanceHints
+ * @group PreloadFonts
+ */
+class Test_MaybeRemoveExistingPreloadedFonts extends TestCase
+{
+
+	protected $config;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->unregisterAllCallbacksExcept( 'rocket_buffer', 'maybe_remove_existing_preloaded_fonts', 10 );
```

Comment:
Method already have a default value of 10: https://github.com/wp-media/wp-rocket/blob/00ed622a2f6ac34448de53136ce5df82171dcb8f/tests/Integration/IsolateHookTrait.php#L56
```suggestion
		$this->unregisterAllCallbacksExcept( 'rocket_buffer', 'maybe_remove_existing_preloaded_fonts' );
```